src/app/components/primary-button.tsx

```diff
@@ -1,9 +1,9 @@
 import { Button, ButtonProps } from '@stacks/ui';
 
 export function PrimaryButton(props: ButtonProps) {
-  const { children, ...rest } = props;
+  const { children, height, ...rest } = props;
   return (
-    <Button borderRadius="10px" height="48px" type="submit" {...rest}>
+    <Button borderRadius="10px" height={height || '48px'} type="submit" {...rest}>
```

Comment:
Without adding this prop, wouldn't a height prop passed in be applied anyway, as you're passing {...rest}?

Reply:
O, so it will override the height set here anyway? I'll try it. 👍

Reply:
Removed and worked so resolving this, thx!